webapp/src/templates/partials/content_row_list_item.html

```diff
@@ -19,8 +19,8 @@ <h4>
       </div>
       <div class="summary">
         <p>{{summary}}</p>
-        <div class="status {{showStatus ? '' : 'ng-hide'}}">
-          <i class="fa {{valid ? 'green-dot' : 'red-dot'}} {{verified ? 'fa-check-circle' : 'fa-circle'}}"></i>
+        <div class="status {{(valid && verified === undefined) ? 'ng-hide' : ''}}">
+          <div class="{{(valid && verified) ? 'mm-check' : 'mm-times'}}"></div>
```

Comment:
Nice one. I think to work with Diana's templates you just need to add some ng-includes here to include the icon you want.